### PR TITLE
recommend heartbeat endpoint status payload, with example

### PR DESCRIPTION
**Pull request by milescrabill**
this formalizes a (good) precedent set by [normandy](https://normandy.services.mozilla.com/__heartbeat__)

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ dispatch.onGet("/__heartbeat__", (req, res) => {
       res.end("Could not find version file")
     } else {
       res.writeHead(200, {"Content-Type":"text/plain"});
```
Comment (mythmon) [inline server.js]: If you're going to be sending back JSON, this should be `Content-Type: application/json`.

Comment (milescrabill) [reply server.js]: Good catch!

**Comment by mostlygeek** [comment]
Nice!  I wish I wrote these into the spec at the beginning.

**Comment by milescrabill** [comment]
I merged this, but due to github weirdness it claims not to have been merged. Closing.

**Pull request closed without merging**